src/qrcore.h

```diff
@@ -27,6 +27,19 @@
 #define __alert(x) QMessageBox::question (this, "Alert", QString(x), QMessageBox::Ok)
 #define __question(x) (QMessageBox::Yes==QMessageBox::question (this, "Alert", QString(x), QMessageBox::Yes| QMessageBox::No))
 
+struct RCoreLocked
```

Comment:
In C++ a class and a struct are the same, with the only difference that `struct` has all member public by default.